scanner/src/lib/mobile-work-machine/DrillManufacturingDataSheet_v0.1.svelte

```diff
@@ -0,0 +1,83 @@
+<script lang="ts">
```

Comment:
This file is clearly failing to follow the obvious previous naming standard for them, even worse than before, this one just randomly gets rid of the `dataproducts` folder too.

![image](https://github.com/ioxiocom/ioxio-tags/assets/1296558/174a155a-4d18-44bb-97bc-bde9554b468d)